src/System.Drawing.Common/src/System/Drawing/Imaging/BitmapData.cs

```diff
@@ -0,0 +1,71 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Drawing.Imaging
+{
+    public partial class BitmapData
+    {
+        /// <summary>
+        /// Specifies the pixel width of the <see cref='Bitmap'/>.
+        /// </summary>
+        public int Width
+        {
+            get { return _width; }
+            set { _width = value; }
+        }
+
+        /// <summary>
+        /// Specifies the pixel height of the <see cref='Bitmap'/>.
+        /// </summary>
+        public int Height
+        {
+            get { return _height; }
+            set { _height = value; }
+        }
+
+        /// <summary>
+        /// Specifies the stride width of the <see cref='Bitmap'/>.
+        /// </summary>
+        public int Stride
+        {
+            get { return _stride; }
+            set { _stride = value; }
+        }
+
+        /// <summary>
+        /// Specifies the format of the pixel information in this <see cref='Bitmap'/>.
+        /// </summary>
+        public PixelFormat PixelFormat
+        {
+            get { return _pixelFormat; }
+            set
+            {
+                if (!Enum.IsDefined(typeof(PixelFormat), value))
```

Comment:
The Windows implementation is cheaper than this, which e.g. will allocate on every call.

Reply:
Thanks, I reverted to the Windows implementation.

Reply:
Thanks